godmine/main.go

```diff
@@ -695,6 +685,81 @@ func editWikiPage(title string) error {
 	return nil
 }
 
+func initConfigFile(endpoint string, apikey string, project string) error {
+	template := `{
+	"endpoint": "%s",
+	"apikey": "%s",
+	"project": %s
+}`
+	content := fmt.Sprintf(template, endpoint, apikey, project)
+
+	file := createConfigFileName()
+	dir := filepath.Dir(file)
+
+	if _, err := os.Stat(dir); err != nil {
+		os.MkdirAll(dir, 0755)
```

Comment:
Sorry this is my bad. Please fix 0700

Reply:
Fixed. 
By the way, MkdirAll/WriteFile with permission what about in Windows os? 
```
$ ls -l ~/.config
....
drwx------   3 user1  staff  102 10 27 20:04 godmine
....
```